charts/osm/values.yaml

```diff
@@ -187,10 +187,10 @@ OpenServiceMesh:
   tracing:
     # -- Toggles Envoy's tracing functionality on/off for all sidecar proxies in the mesh
     enable: false
-    # -- Address of the tracing collector service (must contain the namespace). When left empty, this is computed in helper template to "jaeger.<osm-namespace>.svc.cluster.local". Please override for BYO-tracing as documented in tracing.md
-    address: "jaeger.osm-system.svc.cluster.local"
     # -- Port of the tracing collector service
     port: 9411
+    # -- Address of the tracing collector service (must contain the namespace). When left empty, this is computed in helper template to "jaeger.<osm-namespace>.svc.cluster.local". Please override for BYO-tracing as documented in tracing.md
```

Comment:
nit: you can revert the change in this file

Reply:
I found I cannot revert it. `address` somehow is a required field. Removing it will fail Chart checks

Reply:
looks like you were able to revert. I was not asking to remove the address field. The change you had just moved the lines